pyproject.toml

```diff
@@ -6,3 +6,41 @@ requires = [
   "wheel",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.towncrier]
+package = "molecule"
+package_dir = "molecule"
+filename = "CHANGELOG.rst"
+directory = "changelog/"
+title_format = "molecule {version} ({project_date})"
```

Comment:
Do we need `v` prefix for the version? Some people do this.

Reply:
I am not sure. We have not been exactly consistent on the release conventions. Once the change log is available, it will quickly become clear what should be done. I leave this as-is for now.